lightly/api/api_workflow_upload_embeddings.py

```diff
@@ -79,10 +92,14 @@ def _order_csv_by_filenames(self, path_to_embeddings_csv: str) -> str:
             filenames = [row[index_filenames] for row in rows_without_header]
 
             if len(filenames) != len(self.filenames_on_server):
-                raise ValueError(f"There are {len(filenames)} rows in the embedding file, but "
-                                 f"{len(self.filenames_on_server)} filenames/samples on the server.")
+                raise ValueError(f'There are {len(filenames)} rows in the embedding file, but '
+                                 f'{len(self.filenames_on_server)} filenames/samples on the server.')
             if set(filenames) != set(self.filenames_on_server):
-                raise ValueError(f"The filenames in the embedding file and the filenames on the server do not align")
+                raise ValueError(f'The filenames in the embedding file and the filenames on the server do not align')
+            are_valid_filenames = [_is_valid_filename(f) for f in filenames]
+            if not all(are_valid_filenames):
+                invalid_filenames = [f for f, v in zip(filenames, are_valid_filenames) if v]
+                raise ValueError(f'Invalid filename(s) in embedding file: {invalid_filenames}')
 
```

Comment:
You could make this shorter with
```python
invalid_filenames = [f for f in filenames if not _is_valid_filename(f)]
if len(invalid_filenames) > 0:
    raise ValueError(f'Invalid filename(s) in embedding file: {invalid_filenames}')
```
Furthermore it seems to me that you are getting the **valid, not invalid** filenames in line 101.